content/templates/flatpages/default.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% load staticfiles %}
+
+{% block content %}
+<div class="row">
+	<div class="col-md-12">
+		<div class="maincontent">
+			<h1 class="text-center">{{ flatpage.title }}</h1>
+			<p class="flatpagetext">{{ flatpage.content }}</p>
```

Comment:
if we're allowing custom html inside here, probably better to make this into a `div` instead of a `p`